.stylelintrc.json

```diff
@@ -2,8 +2,30 @@
   "extends": ["stylelint-config-standard"],
   "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
   "rules": {
-    "at-rule-no-unknown": null,
-    "scss/at-rule-no-unknown": true,
+    "at-rule-no-unknown": [
+      true,
+      {
+        "ignoreAtRules": [
+          "tailwind",
+          "apply",
+          "variants",
+          "responsive",
+          "screen"
+        ]
+      }
+    ],
+    "scss/at-rule-no-unknown": [
+      true,
+      {
+        "ignoreAtRules": [
+          "tailwind",
+          "apply",
+          "variants",
+          "responsive",
+          "screen"
+        ]
+      }
+    ],
     "csstree/validator": true
   },
```

Comment:
- [ ] I have Tested your project locally and it's really nice, nice job on using the active storage to attach to the transaction a logo or avatar(jpg, png...), However, your project still needs to follow the styling requirements of the project
![image](https://user-images.githubusercontent.com/65068771/211175700-ca6abfab-813a-45d1-8e58-c642ec26f05d.png) ![image](https://user-images.githubusercontent.com/65068771/211175715-6b5cd73b-ec80-469d-b7c0-4f917ba6b5e3.png) ![image](https://user-images.githubusercontent.com/65068771/211175723-1581c4d9-683b-4c12-9752-ca95bf4655ca.png) ![image](https://user-images.githubusercontent.com/65068771/211175753-6b1a13bc-883d-4110-b86b-be31b446452e.png) ![image](https://user-images.githubusercontent.com/65068771/211175914-fde8ce93-ae13-44f3-8490-cd5d454473e8.png) 

The images above show all the Design requirements you should follow:
![image](https://user-images.githubusercontent.com/65068771/211176025-fed68c67-c16f-48be-93b8-23722cee6f0c.png)